public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>NC News</title>
```

Comment:
Glad to see you've renamed this. 
_technicallly_ I would argue it should be its own commit, and pull request... 
But I will overlook it